packages/hardhat/contracts/PredictorGame.sol

```diff
@@ -21,10 +21,12 @@
 
 // we wont need below line once we finish so will clean once we have the mvp
 import "hardhat/console.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
 
 pragma solidity >=0.8.0 <0.9.0;
 
 contract PredictorGame {
+	using SafeMath for uint256;
```

Comment:
i think as of solidity 0.8x overflow protection is included by default but its not hurting anything and if you want to import the library for future use of the functions that's cool :sunglasses: